Angular/src/app/commentary/commentary.component.html

```diff
@@ -100,6 +100,13 @@ <h6>
     ">
   </ng-container>
   <!-- Show the fragment metrical analysis -->
+  <ng-container
+    *ngTemplateOutlet="
+      show_fragment_content;
+      context: { current_document_content: given_fragment.bibliography, title: 'Bibliography' }
```

Comment:
given_fragment.bibliography voelt wat raar als t in werkelijkheid meer given_fragment.commentary.bibliography is qua data structure, maar tis tenminste wel minder verbose.

Reply:
Klopt. Maar kijk maar eens helemaal bovenaan de code: ik maak een template aan waarin ik het commentaar aanroep met given_fragment wat eigenlijk this.commentary is. Komt omdat ik het er redelijk dom uitgesneden heb bij de migratie naar een eigen component :D Staat op t lijstje.